src/libsync/configfile.h

```diff
@@ -223,6 +223,11 @@ class OWNCLOUDSYNC_EXPORT ConfigFile
     [[nodiscard]] static QString discoveredLegacyConfigPath();
     static void setDiscoveredLegacyConfigPath(const QString &discoveredLegacyConfigPath);
 
+#ifdef Q_OS_MACOS
```

Comment:
let's not make this dependent on the platform
this is not going to cause issues to define this always and removes one `#ifdef`

Reply:
Okay, removed